src/blockchain/src/blockchain.rs

```diff
@@ -191,18 +127,20 @@ mod tests {
         let mut transactions = vec![];
         let data = "Hello First Transaction";
         let transaction = Transaction::new(
-            PartialTransaction::new(TransactionType::Create, local_id, data.as_bytes().to_vec()),
+            TransactionType::Create,
+            local_id,
+            data.as_bytes().to_vec(),
             &keypair,
         );
         transactions.push(transaction);
-        chain.add_block(new_block(
+        chain.add_block(Block::new(
+            chain.blocks()[0].header.hash(),
+            chain.blocks()[0].header.ordinal,
```

Comment:
Applying feedback from https://github.com/pyrsia/pyrsia/pull/517/files/693e1e7b227cac24f141ade09dfa9c0fabbefa8b#r837860004

```suggestion
            chain.blocks()[0].header.ordinal+1,
```